src/common/layout/SSEProvider.tsx

```diff
@@ -65,6 +65,30 @@ export default function SSEProvider({ children }: PropsWithChildren) {
             });
             toast.success(parsedData.message);
             break;
+          case 'pointRanking.attain':
+            queryClient.invalidateQueries({
+              queryKey: userKeys.challenges(1),
+            });
+            toast.success(parsedData.message);
+            break;
+          case 'correctAnswerRanking.attain':
+            queryClient.invalidateQueries({
+              queryKey: userKeys.challenges(1),
+            });
+            toast.success(parsedData.message);
+            break;
+          case 'item.buy':
+            queryClient.invalidateQueries({
+              queryKey: userKeys.challenges(1),
+            });
+            toast.success(parsedData.message);
+            break;
+          case 'progress.updated':
+            queryClient.invalidateQueries({
+              queryKey: userKeys.daily(),
+            });
+            toast.success(parsedData.message);
+            break;
```

Comment:
```suggestion
          case 'pointRanking.attain':
          case 'correctAnswerRanking.attain':
          case 'item.buy':
            queryClient.invalidateQueries({
              queryKey: userKeys.challenges(1),
            });
            toast.success(parsedData.message);
            break;
          case 'progress.updated':
            queryClient.invalidateQueries({
              queryKey: userKeys.daily(),
            });
            toast.success(parsedData.message);
            break;
```
switch case 동일한 로직들은 합쳐주세요

Reply:
오호 이렇게도 가능하군요. 감사합니다!

Reply:
[a975e84](https://github.com/modern-agile-team/8term-coko-Front/pull/201/commits/a975e84fc29ae4fbb41353198e27d7e39b8b8c31)

해당 커밋에서 수정하였습니다~